TypeScript.py

```diff
@@ -1,19 +1,13 @@
 import sys
+import os
 
-# Sublime/Python 2 & 3 differ in the name of this module, thus package import
-# needs to be handled slightly differently
-if sys.version_info < (3, 0):
-    from libs import *
-    from libs.reference import *
-    from libs.view_helpers import *
-    from listeners import *
-    from commands import *
-else:
-    from .libs import *
-    from .libs.reference import *
-    from .libs.view_helpers import *
-    from .listeners import *
-    from .commands import *
+sys.path.append(os.path.dirname(os.path.abspath(__file__)))
+
+from typescript_module.libs import *
+from typescript_module.libs.reference import *
+from typescript_module.libs.view_helpers import *
+from typescript_module.listeners import *
+from typescript_module.commands import *
```

Comment:
Not big on the name `typescript_module`; what about `typescript_sublime` or just `typescript`?